cli/update.ml

```diff
@@ -15,7 +15,9 @@ let update ~total ~updated src_dep =
   let open Result.O in
   let ref : Duniverse.resolved = src_dep.Duniverse.Deps.Source.ref in
   incr total;
-  Exec.git_resolve ~remote:src_dep.upstream ~ref:ref.t >>= fun new_ref ->
+  Cache.update ~remote:src_dep.upstream ~ref:ref.t ()
```

Comment:
I don't think that does the right thing either, this seems to update the cache, which is not what we want here. We want to update the commit hash corresponding to the tracked ref in the dune-get.